pylink/jlink.py

```diff
@@ -545,6 +545,12 @@ def open(self, serial_no=None, ip_addr=None):
 
         if ip_addr is not None:
             addr, port = ip_addr.split(':')
+            if addr == 'tunnel':
```

Comment:
I think here we want to handle the possibility that the user gave us `tunnel:<serial_no>:<port>` as that is the standard.  It would make it a bit more complicated though.